config.js

```diff
@@ -92,6 +136,10 @@ module.exports = {
           format: 'css/variables',
           filter: 'isDark',
         },
+        {
+          destination: 'tokens.module.css',
+          format: 'splitByMode',
+        },
```

Comment:
This outputs variables into one file split by `.dark` and `.light`. Then in the `Theme` component we can easily switch between classnames